src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1677,6 +1681,26 @@ class DocBaseViewer extends BaseViewer {
         // in which the default discoverability experience is enabled
         this.containerEl.setAttribute(DISCOVERABILITY_ATTRIBUTE, isUsingDiscoverability);
     }
+
+    /**
+     * Hides the create region cursor popup for a document
+     *
+     * @protected
+     * @return {void}
+     */
+    applyCursorFtux() {
+        if (!this.containerEl) {
+            return;
+        }
+
+        if (this.annotationControlsFSM.getState() === AnnotationState.REGION) {
+            if (this.cache.get(DOCUMENT_FTUX_CURSOR_SEEN_KEY)) {
+                this.containerEl.classList.add(CLASS_ANNOTATIONS_DOCUMENT_FTUX_CURSOR_SEEN);
```

Comment:
Is there any chance this class won't get applied by the time the `PopupCursor` is already mounted in box-annotations and we might see a flash of the popup?

Reply:
Discussed offline, it looks like this issue shouldn't happen.